### ISSUE TITLE
disable-create-button patch fails in YouTube 17.24.34

### ISSUE THREAD

**Issue by shivanshkverma** [revanced/revanced-patches#52]
![image](https://user-images.githubusercontent.com/15206857/175238541-b40a7ded-47d3-42a6-8a8a-3e6005e9ed35.png)
Command that I used: `java -jar revanced-cli.jar -a youtube.apk -m app-release-unsigned.apk -o revanced.apk -d f43f18c2 -b revanced-patches.jar`
Patches version: v1.10.0
CLI version: v1.11.0
Integrations: v0.13.0
Version: non-root

Also, the version is not updated in prerequisites.

**Comment by TheJeterLP** [comment]
Probably https://github.com/revanced/revanced-patches/commit/314e955a3de0a95ef1679ba1e487d29f7ca4c541 is the problem here.

**Comment by TheJeterLP** [comment]
@oSumAtrIX #45 is not in the newest release. this probably fixes the error.

**Comment by oSumAtrIX** [comment]
@TheJeterLP He was using the latest files

Patches version: v1.10.0
CLI version: v1.11.0
Integrations: v0.13.0
Version: non-root

**Comment by TheJeterLP** [comment]
> @TheJeterLP He was using the latest files
> 
> Patches version: v1.10.0 CLI version: v1.11.0 Integrations: v0.13.0 Version: non-root

I know that, but #45 is not integrated in the latest release because it did not trigger a release :)

**Comment by TheJeterLP** [comment]
1.10.0 only contains my update PR, but not #45 which is now needed by CLI.

**Comment by oSumAtrIX** [comment]
#45 has already been merged

**Comment by TheJeterLP** [comment]
> #45 has already been merged

Yes. Read what I say. The merge of #45 did NOT trigger a new release! That's why 1.10.0 does not contain the changes of #45! 1.10.0 release got created before #45 got merged!

**Comment by oSumAtrIX** [comment]
Then at the time where #45 was not merged, #49 was incompatible. Test your changes against HEAD before PRing to avoid issues like this

**Comment by TheJeterLP** [comment]
> Then at the time where #45 was not merged, #49 was incompatible. Test your changes against HEAD before PRing to avoid issues like this

I did. The current CLI version relies on the new methods used in #45. So you released a new CLI version that's incompatible to patches.

**Comment by oSumAtrIX** [comment]
#45 does not introduce breaking changes. The patcher supports both APIs, the one before and the one after #45.

**Comment by oSumAtrIX** [comment]
Closing in favor of #56.
